app/routes.py

```diff
@@ -1,2 +1,289 @@
-from flask import Blueprint
+from app import db
+from app.models.task import Task
+from app.models.goal import Goal
+from flask import Blueprint, jsonify, request
+import requests
+from datetime import datetime
+import os
+from dotenv import load_dotenv
 
+from tests.test_wave_06 import test_post_task_ids_to_goal
+
+tasks_bp = Blueprint("tasks", __name__, url_prefix="/tasks")
+goals_bp = Blueprint("goals", __name__, url_prefix="/goals")
+
+
+@tasks_bp.route("", methods=["POST"])
```

Comment:
👍 

Nice job splitting your endpoints into separate functions. There are still shared pieces of functionality in some of the endpoints that we can think about trying to reduce duplication (especially looking up a record by its id), but on the plus side, in each function, we only need to think about the logic related to that specific endpoint.